package.json

```diff
@@ -326,6 +326,7 @@
     "coffee-script": "^1.10.0",
     "coveralls": "^2.11.15",
     "cross-spawn": "^5.1.0",
+    "del": "^3.0.0",
```

Comment:
what's this for?

Reply:
In the module `options.spec.js` (see lines `:58` and `:66`), a test data was created and `del` is used to remove it before the running test concludes.